code/game/objects/effects/anomalies.dm

```diff
@@ -31,10 +31,13 @@
 
 
 /obj/effect/anomaly/proc/anomalyNeutralize()
-	new /obj/effect/effect/bad_smoke(loc)
+	var/turf/T = get_turf(src)
 
-	for(var/atom/movable/O in src)
-		O.loc = src.loc
+	new /obj/effect/effect/bad_smoke(T)
+
+	if(aSignal)
+		aSignal.forceMove(T)
+		aSignal = null
```

Comment:
Or I could make it so when an anomaly is deleted by any means, it just moves the core out.

🤔 

not sure which method to go with.